platform/darwin/src/MGLTypes.h

```diff
@@ -95,10 +95,15 @@ typedef struct __attribute__((objc_boxable)) MGLTransition {
      The amount of time in seconds to wait before beginning the animation.
      */
     NSTimeInterval delay;
+    
+    /**
+     A boolean indicating whether label placement transitions are enabled.
+     */
+    BOOL enablePlacementTransitions;
```

Comment:
The following two syntaxes are valid and common ways of creating a struct like `MGLTransition` in Objective-C and Objective-C++ code:

```objc
MGLTransition transition = (MGLTransitionMake) {1, 2};
MGLTransition transition = (MGLTransitionMake) { .duration = 1, .delay = 2 };
```

With this change, those initializer lists produce the following compiler warning:

```
Missing field 'enablePlacementTransitions' initializer
```

A compiler warning alone may or may not be considered a backwards-incompatible change. After all, enabling `-Wall` or `-Weverything` is practically asking for minor changes when upgrading dependencies. However, Swift code will wind up with a compiler error (with a fix-it suggestion):

```swift
let transition = MGLTransition(duration: 1.0, delay: 2.0)
```

```
Missing argument for parameter 'enablePlacementTransitions' in call
```